src/helpers/http_client.py

```diff
@@ -0,0 +1,132 @@
+import requests
+from src.helpers.counters import execution_time_delta
+
+
+class SessionHttp(requests.Session):
+    def __init__(self):
+        super().__init__()
+
+    def autorization(self):
+        pass
+
+    def get_request(self, *args, **kwargs):
+        r"""Sends a GET request.
+        :param url: URL for the new :class:`Request` object.
+        :param params: (optional) Dictionary, list of tuples or bytes to send
+            in the query string for the :class:`Request`.
+        :param \*\*kwargs: Optional arguments that ``request`` takes.
+        :return: :class:`Response <Response>` object
+        :rtype: requests.Response
+        """
+        return self.get(*args, **kwargs)
+
+    def post_request(self, *args, **kwargs):
+        r"""Sends a POST request.
+        :param url: URL for the new :class:`Request` object.
+        :param data: (optional) Dictionary, list of tuples, bytes, or file-like
+            object to send in the body of the :class:`Request`.
+        :param json: (optional) json data to send in the body of the :class:`Request`.
+        :param \*\*kwargs: Optional arguments that ``request`` takes.
+        :return: :class:`Response <Response>` object
+        :rtype: requests.Response
+        """
+        return self.post(*args, **kwargs)
+
+    def put_request(self, *args, **kwargs):
+        r"""Sends a PUT request.
+        :param session: (Optional)  :default value False:
+        Boolean type param if true request is sending in session, None or False sending 1 request 1 session.
+        :param url: URL for the new :class:`Request` object.
+        :param data: (optional) Dictionary, list of tuples, bytes, or file-like
+            object to send in the body of the :class:`Request`.
+        :param json: (optional) json data to send in the body of the :class:`Request`.
+        :param \*\*kwargs: Optional arguments that ``request`` takes.
+        :return: :class:`Response <Response>` object
+        :rtype: requests.Response
+        """
+        return self.put(*args, **kwargs)
+
+    def delete_request(self, *args, **kwargs):
+        r"""Sends a DELETE request.
+        :param session: (Optional)  :default value False:
+        Boolean type param if true request is sending in session, None or False sending 1 request 1 session.
+        :param url: URL for the new :class:`Request` object.
+        :param \*\*kwargs: Optional arguments that ``request`` takes.
+        :return: :class:`Response <Response>` object
+        :rtype: requests.Response
+        """
+        return self.delete(*args, **kwargs)
+
+
+class ClientHttp:
+    """Class for implementing http requests logic"""
+
+    def __init__(self):
+        self.request = requests
+        self.authorization = None
+
+    def get_request(self, *args, **kwargs):
+        r"""Sends a GET request.
+        :param url: URL for the new :class:`Request` object.
```

Comment:
Your docstrings parameters don't correspond to the parameters in your method. Relevant to all methods in this class